src/modules/download.ts

```diff
@@ -326,10 +326,12 @@ export class DownloadManager extends EventEmitter {
                 try {
                     const stats = await fs.stat(absolutePath)
                     if (
-                        stats.mtime.getTime() / 1000 !== file.timemodified
-                        || stats.size !== file.filesize
+                        file.filesize !== 0 &&
+                        (stats.mtime.getTime() / 1000 !== file.timemodified
+                            || stats.size !== file.filesize)
```

Comment:
is there any reason for not checking the timemodified when the filesize is 0? seems like the timemodified is still good, otherwise no updates to the file will ever be noticed by WeBeep Sync
```suggestion
                        stats.mtime.getTime() / 1000 !== file.timemodified
                        || (file.filesize !== 0 && stats.size !== file.filesize)
```
this way only the size check will be skipped if the size is wrong